build.properties

```diff
@@ -49,8 +49,8 @@ marketplace.metadata.development_stage.phase=2
 # project.revision=7.0-17.02.28
 
 # # ivy properties
-# dependency.pentaho-cpk-plugin.revision=7.0.0.0-25
-# dependency.pentaho-cpf-plugin.revision=7.0.0.0-25
+dependency.pentaho-cpk-plugin.revision=9.1.0.2-344
+dependency.pentaho-cpf-plugin.revision=9.1.0.0-101

```

Comment:
I think we should create multiple branches for 8.3, 9.0 and 9.1 in this repository.
In here we should put: 8.3.0.0-371, 9.0.0.0-423 and 9.1.0.0-324 for each respective branch.

Also, https://github.com/pentaho/marketplace-metadata/blob/master/marketplace.xml should be updated to make this plugin available for 8.3, 9.0 and 9.1 versions of Pentaho Server and new versions of this plugin should be built and made available somewhere so the file can be downloaded.

I guess three new blocks like this https://github.com/pentaho/marketplace-metadata/blob/master/marketplace.xml#L1655-L1670 need to be added.

@pamval, @graimundo and @diogofscmariano What is your take on this?

Reply:
For the master branch, should we use 9.2-SNAPSHOT or keep the 9.1 artifact?